src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1075,6 +1131,20 @@ public void cleanUpTempConfig() {
         }
     }
 
+    public void cleanUpTempDockerfile() {
+        if (tempDockerfilePath != null) {
+            File tempDockerfile = tempDockerfilePath.toFile();
+            if (tempDockerfile.exists()) {
+                try {
+                    Files.delete(tempDockerfilePath);
+                    debug("Sucessfully deleted dev mode temporary Dockerfile");
+                } catch (IOException e) {
+                    error("Could not delete dev mode temporary Dockerfile");
```

Comment:
Include the exception message so there is some indication on why it couldn't delete.

Reply:
I'm not sure why it wouldn't be able to be deleted, other than it not existing (which we check for). I just mirrored this function based off of `cleanUpTempConfig()`

Reply:
It's fine as is, but having the error included would be better for diagnosis (e.g. maybe if the file was somehow still in use).

Reply:
I suggest using e.getMessage() in case it has an explanation.